services/pom.xml

```diff
@@ -52,7 +55,7 @@
         <dependency>
             <groupId>org.keycloak</groupId>
             <artifactId>${keycloak.crypto.artifactId}</artifactId>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
I think this should not be done as it can break our FIPS integration. The `keycloak-services` module should not have any direct dependencies on BouncyCastle, but all the crypto operations should be done with the `CryptoProvider` interface if needed.

But on the other hand, I can see that you're using only in the test class `SigningServiceTest`, so maybe this is not needed?

Reply:
I set it back to test, this was by mistake. Fixed in [19e0371](https://github.com/keycloak/keycloak/pull/26484/commits/19e037112f0cd49abbb80b9544d49f74bc9ade7e)

Reply:
Thank! Resolving this point